docs/configuration.rst

```diff
@@ -69,8 +69,34 @@ And configure its URL:
     # StatsD
     cliquet.statsd_url = udp://carbon.server:8125
 
+The following counters will be enabled:
 
-In order to enable *Heka* logging output:
+* ``users`` (unique user ids)
+* ``authn_type.basicauth``
+* ``authn_type.fxa``
+
+And the following timers:
+
+* ``authentication.permits``
+* ``view.hello.GET``
+* ``view.heartbeat.GET``
+* ``view.batch.POST``
+* ``view.bucket-[record|collection].[GET|POST|PUT|PATCH|DELETE]``
```

Comment:
I would do: 

 - view.bucket-collection.[GET|POST|PUT|PATCH|DELETE]
 - view.bucket-record.[GET|POST|PUT|PATCH|DELETE]

One pattern matching is enough for human brain :)